content/reference-rippled.md

```diff
@@ -3012,7 +3012,7 @@ The response follows the [standard format](#response-formatting), with a success
 | account\_id | String | The [Address][] of the account. |
 | public\_key | String | The public key of the account, in encoded string format. |
 | public\_key\_hex | String | The public key of the account, in hex format. |
-{#| warning | String | (May be omitted) If the request specified a seed value, this field provides a warning that it may be insecure. TODO: uncomment when this gets added in 0.32.0 |#}
+| warning | String | (May be omitted) If the request specified a seed value, this field provides a warning that it may be insecure. _(New in [version 0.32.0][])_ |
```

Comment:
* wallet_propose:
    1. Fixed 17 cases by find/replace
    2. Yes, the format is uniform for all 17 admin commands
    3. Good idea. Added.
* Actually, I realized this whole section changed again with 0.32.0 so I needed to reorganize it. The commandline syntax doesn't support the "seed" or "seed_hex" fields, either, but that's not a limitation in functionality since the passphrase field can cover those values. I joined the Ed25519 experimental note and the comment that the commandline version can't generate Ed25519 keys together.
* "Specifying a Seed" -
    1. The problem is that our style actually does "handle" level 5 headers, insomuch as it explicitly styles them to look a lot like paragraph text. I can raise with Ben.
    2. Agreed, it makes sense to just make this text instead of a callout. Done.